invenio/base/config.py

```diff
@@ -74,7 +75,6 @@
     'invenio.modules.comments',
     'invenio.modules.communities',  # remove with invenio/modules/communities
     'invenio.modules.multimedia',
-    'invenio.modules.pages',
```

Comment:
I can keep it, what is the use of it when the module is completely removed from Invenio?